src/components/AdminForm.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { connect } from "react-redux";
+import { setCohortName, setCohortType } from "../redux/actions/adminFormActions";
+
+import TextInput from "./TextInput";
+import Select from "./Select";
+import SubmitButton from "./SubmitButton";
+
+const AdminForm = (props) => {
```

Comment:
you can destructureing props here to be consistent with all other components

Reply:
Not using destructuring here as this component already has 4 props, and the general recommendation is to use destructuring in props if it's 3 or less. (otherwise the line becomes too long and unruly).

We can discuss how to deal with cases with 4 or more props later as a group to make a common style guide decision.